packages/airnode-admin/src/implementation.ts

```diff
@@ -30,7 +30,7 @@ export const deriveWalletPathFromSponsorAddress = (sponsorAddress: string): stri
     const shiftedSponsorAddressBN = sponsorAddressBN.shr(31 * i);
     paths.push(shiftedSponsorAddressBN.mask(31).toString());
   }
-  return `0/${paths.join('/')}`;
+  return `1/${paths.join('/')}`;
```

Comment:
Might as well update the comment in line 21

Reply:
Thanks. I was sure I got them all... Not sure how this one slipped :/